### PR TITLE
Two fixes for RTAMO

### PR DESCRIPTION
**Pull request by willdurand**
Fixes https://github.com/mozilla/addons-frontend/issues/10469
Fixes https://github.com/mozilla/addons-frontend/issues/10470

---

I tried not to make too many changes to avoid conflicts when we'll cherry-pick the commits (they should apply cleanly right now).

### CODE REVIEW

**Comment by willdurand** [comment]
(I added many reviewers to have several pairs of eyes but feel free to skip the request)

File: tests/unit/amo/components/TestGetFirefoxButton.js [hunk 1]
```diff
@@ -135,33 +134,48 @@ describe(__filename, () => {
         });
       });
 
-      it.each([true, false])(
```
Comment (bobsilverberg) [inline tests/unit/amo/components/TestGetFirefoxButton.js:138]: Why did you remove the `it.each` and replace it with two separate test cases that do the same thing? Can't we just use `it.each` here and add a comment explaining why there is no variation in the expectations?

**Review by bobsilverberg — APPROVED** [review]
It looks good, but I have a question about a test, and I'm not sure we actually need to disable the experiment. We should possibly discuss that further.

r+wc

**Comment by bobsilverberg** [comment]
Because this needs to be cherry-picked and then tested for today's push, I'm going to merge this as is.

File: tests/unit/amo/components/TestGetFirefoxButton.js (hunk 1 above)
Comment (willdurand) [reply tests/unit/amo/components/TestGetFirefoxButton.js:138]: ah yes, I removed one of the cases and then re-added it later..
